### ISSUE TITLE
 Error: Element for MapView already registered

### ISSUE THREAD

**Issue by camilo02021** [nativescript-vue/nativescript-vue#1037]
Issue when I use "$navigateTo" when click on button I got " Error: Element for MapView already registered." It is because I'm using the plugging like this: 
"import GoogleMaps from '@nativescript/google-maps/vue'
 const app = createApp(Home)
 app.use(GoogleMaps).start()"
It is how the documentation says, I think the "use.()" and the "$navigateTo" they conflict.
Any solution for this?

**Comment by vallemar** [comment]
@camilo02021 you can register plugin as: https://github.com/nativescript-vue/nativescript-vue/issues/1018#issuecomment-1502351847

**Comment by camilo02021** [comment]
Thank you it works, it happens only with vue3, In vue2 I can use it by "Vue.use(GoogleMaps)"

**Comment by camilo02021** [comment]
Thank you Juan, I appreciate your help

**Comment by vallemar** [comment]
@camilo02021 Yes, this is related to navigation. In the meantime you can declare your components like this, it's totally fine. I think it will be solved with this PR https://github.com/nativescript-vue/nativescript-vue/pull/1036
